src/components/Personalization/dom-actions/clicks/collectClicks.js

```diff
@@ -19,31 +19,76 @@ const getMetasIfMatches = (
 ) => {
   const { documentElement } = document;
   let element = clickedElement;
+  let i = 0;
 
   while (element && element !== documentElement) {
     if (matchesSelectorWithEq(selector, element)) {
-      return getClickMetasBySelector(selector);
+      const matchedMetas = getClickMetasBySelector(selector);
```

Comment:
Can we just extract the first label found? 
As far as I saw in the design wiki we should pick up the first label from one of the matching selectors.

Reply:
We can't just extract the first label, since we don't know for sure whether it belongs to the innermost DOM element, or whether it's higher up the DOM tree. This is mentioned explicitly in the design wiki:
> In case of multiple click-tracking selector matches, all matching propositions will be included in a single interact event, with the event propositionAction → label set to the label of the first click-tracking proposition having a label, **when traversing the DOM tree from the innermost to the outermost DOM element**.